stock_prices/stock_prices.py

```diff
@@ -3,13 +3,18 @@
 import argparse
 
 def find_max_profit(prices):
-  pass
+    profit = min(prices) - max(prices)
+    for buy_index in range(len(prices)):
+        for sell_index in range(buy_index + 1, len(prices)):
```

Comment:
this works great! however, it's a double nested for loop, making your solution O(n**2). after you're done with all of MVP, see if you can bring this down to O(n)